### ISSUE TITLE
Question on 10x Genomics data

### ISSUE THREAD

**Issue by YingChen94** [bcgsc/physlr#193]
Hello, 

Thank you for making this tool! I have been struggling with supernova assembly from 10x data due to big genome size (4Gb). Does physlr have any requirements in terms of computer resources for such big genome size? For building physical maps, is the input file the raw sequencing data? Or do I need to do any pre-processing? I am thinking to use a different species in the same genus as the reference genome. Is it appropriate? 

Thank you!
Ying

**Comment by aafshinfard** [comment]
Hello @YingChen94  and thank you for opening an issue,
The input from 10x needs to be preprocessed with LongRanger and the barcodes should be in the read header in the tag format BX:Z common for 10x reads. Our experiments on Humans showed ram requirements of up to 200 GB but it does not necessarily scale up with the size of the genome and depends on the coverage too. I previously ran it successfully on a plant genome of roughly that size but the physical maps were not very contiguous.
Physlr is a _de novo_ tool and does not require a reference genome but can accept one for evaluation purposes. If you trust the reference genome is closely related you may use that to get some sense on the quality of the results.

**Comment by YingChen94** [comment]
Thank you for fast reply! Do I need to filter those sequences without the "BX:Z" barcode (I think those reads have invalid barcodes) before running Physlr?

**Comment by aafshinfard** [comment]
Yes @YingChen94 , I believe you need to filter them before running Physlr. Does Longranger not filter them by default or by adding a flag?

**Comment by YingChen94** [comment]
Hi Amirhossein, the output from Longranger includes reads without "BX:Z" barcode flag. I am using Physlr v1.0.4 and it seems to have ran the first step histogram successfully.
